docs/docsite/rst/getting_started/get_started_playbook.rst

```diff
@@ -42,7 +42,7 @@ In this output you can see:
 * The names that you give the play and each task.
   You should always use descriptive names that make it easy to verify and troubleshoot playbooks.
 
-* The ``Gather Facts`` task runs implicitly.
+* The "Gathering Facts" task runs implicitly.
```

Comment:
```suggestion
* The ``Gathering Facts`` task runs implicitly.
```

Reply:
@samccann May I ask why you changed this back after we discussed it here: https://github.com/ansible/ansible-documentation/pull/1175#discussion_r1513371225 ?

Reply:
Because I'm an idiot who didn't read the comment?  I can patch in a separate pr. sorry about that.

Reply:
OK, no worries!